src/plugins/home/public/plugin.ts

```diff
@@ -168,6 +174,36 @@ export class HomePublicPlugin
       workspaceAvailability: WorkspaceAvailability.outsideWorkspace,
     });
 
+    if (core.chrome.navGroup.getNavGroupEnabled()) {
```

Comment:
Nit: Not sure if it is suitable to register the page inside home plugin.

Reply:
agree, It's a little wired we put into home plugin, we might need to have a new plugin to store use case overview pages.